### PR TITLE
Api

### PR DESCRIPTION
**Pull request by AL9000**

### CODE REVIEW

**Comment by AL9000** [comment]
@SebCorbin Is it usefull yet to add pagination?

File: api/views.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+from history.models import HistoryEntry
+
+__all__ = [
+    "HistoryEntryListAPIView",
+]
+
+
+class HistoryEntryListAPIView(ListAPIView):
+    serializer_class = HistoryEntryListSerializer
+    authentication_classes = [SessionAuthentication, ]
```
Comment (SebCorbin) [inline api/views.py]: Do we open history entries to everyone (question asked in chat)?

**Comment by SebCorbin** [comment]
Let's not paginate at the moment, even with 100 entries, it should be fast enough (and pagination would be hard on timelines)

File: api/serializers.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+]
+
+
+class HistoryEntryListSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = HistoryEntry
+        fields = [
+            "message",
+            "created",
+        ]
```
Comment (AL9000) [inline api/serializers.py]: @SebCorbin I removed the object's uuid from here, it's not needed as we already know it :)

**Review by SebCorbin — APPROVED** [review]
Good for me for now, we can handle the permissions question later (via a mixin maybe)

File: api/views.py (hunk 1 above)
Comment (SebCorbin) [reply api/views.py]: > Pour les entrées d'historique, chacun doit implémenter sa propre vue (qui héritera d'une vue généraliste dans contrib\views.py) ==> Il faudra juste redéfinir la permission

**Pull request merged**